doc/dev/quick_start.rst

```diff
@@ -249,9 +249,9 @@ are supported in Catalyst, although not all features are supported for all measu
    :header-rows: 0
 
    * - :func:`qml.expval() <pennylane.expval>`
-     - The expectation value of all observables is supported.
+     - The expectation value of all observables and finite-shots are supported.
    * - :func:`qml.var() <pennylane.var>`
-     - The variance of all observables is supported.
+     - The variance of all observables and finite-shots are supported.
    * - :func:`qml.sample() <pennylane.sample>`
      - Samples in the computational basis only are supported.
    * - :func:`qml.counts() <pennylane.counts>`
```

Comment:
Does this unlock finite shots for probs as well?

Reply:
yes, I've added it to README of the runtime. But, we didn't have any mention of this unsupported feature for `probs` in the quick start demo, so didn't update anything here.